skyportal/tests/api/test_photometric_series.py

```diff
@@ -0,0 +1,2176 @@
+import os
+import time
+import hashlib
+import base64
+import pytest
+import uuid
+
+import numpy as np
+import pandas as pd
+
+from astropy.time import Time
+
+import sqlalchemy as sa
+from sqlalchemy.exc import IntegrityError
+
+from skyportal.tests import api, assert_api, assert_api_fail
+from skyportal.models import DBSession, PhotometricSeries
+
+
+def convert_dataframe_to_bytes(df, metadata=None, encode=True):
```

Comment:
seems like a broadly useful function here. Can you put it in utils.py and then import it for these tests?